crates/uv/src/commands/pip/uninstall.rs

```diff
@@ -49,12 +47,11 @@ pub(crate) async fn pip_uninstall(
     let spec = RequirementsSpecification::from_simple_sources(sources, &client_builder).await?;
 
     // Detect the current Python interpreter.
-    let environment = PythonEnvironment::from_toolchain(Toolchain::find(
+    let environment = PythonEnvironment::find(
         python.as_deref().map(ToolchainRequest::parse),
         EnvironmentPreference::from_system_flag(system, true),
-        ToolchainPreference::from_settings(preview),
         &cache,
-    )?);
+    )?;
```

Comment:
This is the gist of the change. We want an environment, not an installed toolchain. We don't need to specify toolchain preferences.